packages/EasyRandom/readme.md

```diff
@@ -71,6 +71,18 @@ $randomGenerator = (new \EonX\EasyRandom\RandomGenerator())->setUuidV4Generator(
 $uuidV4 = $randomGenerator->uuidV4();
 ```
 
+#### In Symfony configure your preferred UUID V4 generator
+
+To make this possible, the bundle will register the built-in generators as services.
+
+- symfony_uuid4
```

Comment:
I think here we should put the full alias so it makes sense to the readers why in the following example we're using `easy_random.symfony_uuid4`